src/Admin/MultisitesModelAdminExtension.php

```diff
@@ -27,24 +27,25 @@ class MultisitesModelAdminExtension extends Extension {
 	 * @var DataObject
 	 **/
 	private $listDataClass;
-	
+
 	/**
 	 * If this dataClass is MultisitesAware, set the Multisites_ActiveSite 
 	 * session variable to one of the follwing:
 	 * a) The SiteID passed in the request params, if it exists
 	 * b) The current site, if the current member is a manager of that site
 	 * c) The first site that the current member is a manager of
 	 **/
-	public function onAfterInit(){	
-		if($this->modelIsMultiSitesAware()) {
+	public function onAfterInit(){
+		$this->preventMultisitesAware();
```

Comment:
Any chance this could be named something different? This implies that it *always* prevents multisites aware, where it's actually more of a check and prevent.

Reply:
Done.